src/main/java/core/basesyntax/MyLinkedList.java

```diff
@@ -3,48 +3,160 @@
 import java.util.List;
 
 public class MyLinkedList<T> implements MyLinkedListInterface<T> {
+    private Node<T> head; //початок
+    private Node<T> tail; //кінець
+    private int size = 0;
+
+    private static class Node<T> {
+        private T value;
+        private Node<T> next;
+        private Node<T> prev;
+
+        public Node(Node<T> prev, T value, Node<T> next) {
+            this.value = value;
+            this.next = next;
+            this.prev = prev;
+        }
+    }
+
     @Override
     public boolean add(T value) {
-        return false;
+        if (size == 0) {
+            head = new Node<>(null, value, null);
+            tail = head;
+            size++;
+            return true;
+        }
+        Node<T> lastTail = tail;
+        Node<T> newTail = new Node<>(lastTail, value, null);
+        lastTail.next = newTail;
+        tail = newTail;
+        size++;
+        return true;
```

Comment:
good, but let's simplify it a bit:
```suggestion
        Node<T> newNode = new Node<>(tail, value, null);
        if (size == 0) {
            head = newNode;
        } else {
            tail.next = newNode;
        }
        tail = newNode;
        size++;
        return true;
```

Reply:
done